CHANGELOG.md

```diff
@@ -20,6 +20,20 @@ Visit The National Archives' [documentation on OGL and Crown copyright](https://
 
 This was added in [pull request #2702: Allow localisation of content licence and copyright notices in Footer](https://github.com/alphagov/govuk-frontend/pull/2702).
 
+#### Pass HTML directly into compatible components
+
+If using the Nunjucks macros, you can now pass HTML content directly into compatible components using [the Nunjucks `call` syntax](https://mozilla.github.io/nunjucks/templating.html#call). In these instances, the called HTML is treated the same as the `html` parameter.
+
+Components that have been updated to support this syntax are:
+
+- Details
+- Error summary (mapped to `descriptionHtml` parameter)
```

Comment:
I'm now very superstitious about the Error Summary and any changes that might affect screenreaders, but I don't think there should be any issue (from our side) on this change.

Reply:
The resulting HTML from the Nunjucks changes shouldn't be different between the call syntax and the `html` parameter, so I can't imagine there'll be a problem. Can always give @owenatgov a poke so he has to think about the error summary for a bit longer, though. 😜

Reply:
Don't you put that evil on him!

Reply:
NO MORE.

In all seriousness this looks fine to me. The problems with the error summary I'm pretty sure are how our focusing on load via js is interacting with screen readers and less so the markup. This will impact my proposed error summary change, i it goes in, but that's something for me to deal during the rebase process.